### DISCUSSION TITLE
Decoding (understanding) OPT and NEWOPT values at reset

### DISCUSSION THREAD

**Discussion by madman-011235813** [grblHAL/iMXRT1062#79] [Q&A] [answered]
Hello.

I am compiling grblHAL for my Teensy board with an experimental feature and would like to ensure that the settings that I choose in my_machine.h match the settings from when I built months ago using the Web Builder. I am hoping I can extract what I need from the "OPT" and "NEWOPT" output that is provided when the board reboots.

```
[VER:1.1f.20230427:]
[OPT: VNMHSL2,35,1024,3,0]
[AXS:3:XYZ]
[NEWOPT:ENUMS,RT+,HOME,ES,REBOOT,TC,SED,RTC,ETH]
...
```

Can anyone direct me to a resource that will help me decode this? 

Thanks in advance.

**Comment by terjeio** [comment, marked as answer]
See the [Wiki](https://github.com/grblHAL/core/wiki/Report-extensions#system-command-extensions) - and also take not of any plugins that is reported.
